apps/src/applab/rgbcolor.js

```diff
@@ -231,11 +245,15 @@ module.exports = function(color_string)
     this.r = (this.r < 0 || isNaN(this.r)) ? 0 : ((this.r > 255) ? 255 : this.r);
     this.g = (this.g < 0 || isNaN(this.g)) ? 0 : ((this.g > 255) ? 255 : this.g);
     this.b = (this.b < 0 || isNaN(this.b)) ? 0 : ((this.b > 255) ? 255 : this.b);
+    this.a = (this.a < 0) ? 0 : ((this.b > 1 || isNaN(this.a)) ? 1 : this.a);
```

Comment:
Should replace `this.b > 1` with `this.a > 1` on this line?  Otherwise color that includes blue is opaque?